Cargo.lock

```diff
@@ -1302,6 +1351,39 @@ version = "0.2.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "2839e79665f131bdb5782e51f2c6c9599c133c6098982a54c794358bf432529c"
 
+[[package]]
+name = "openssl"
+version = "0.10.30"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "8d575eff3665419f9b83678ff2815858ad9d11567e082f5ac1814baba4e2bcb4"
+dependencies = [
+ "bitflags",
```

Comment:
Yikes, that's bad, we don't want to add open ssl as a dependency

Reply:
hmm not sure what we can do about that... we cant just disable tls for ws :/

Reply:
I can take some time next week to add rustls to tungestenite

Reply:
That wont help much for quite a while; we use tokio_tungstenite version that warp uses, which already is an older version, so until that happens...

Reply:
> That wont help much for a while; we use the tokio_tungstenite version that warp uses, which already is an older version, so until that happens...

Not a big deal, `warp` is used only as a dev dependency, so it's fine if there's a duplicate there. Just not in the final binary.

Reply:
Fixed by manually establishing the TLS connection instead of letting tokio_tungstenite do it.